src/providers/BasicPoolsProvider.tsx

```diff
@@ -321,12 +299,12 @@ export async function getPoolsDataFromRegistry(
 }
 
 function buildMetaInfo(
-  sharedSwapData: DeepNullable<SharedSwapData>,
```

Comment:
seems like we don't actually need DeepNullable, or at least we don't need it here since you already did null checks before this fn